perllib/FixMyStreet/Script/SendStuckReportsSummary.pm

```diff
@@ -0,0 +1,59 @@
+package FixMyStreet::Script::SendStuckReportsSummary;
+
+use v5.14;
+use warnings;
+
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use Lingua::EN::Inflect qw(PL_N PL_V WORDLIST);
+
+sub run {
+    my $params = shift;
+    my $cobrand = $params->{body}->get_cobrand_handler;
+
+    my @stuck_reports = FixMyStreet::DB->resultset('Problem')->to_body($params->{body}->id)->search({
+        category => $params->{categories},
+        send_state => 'unprocessed',
+        state => [ FixMyStreet::DB::Result::Problem::open_states() ],
+        send_fail_count => { '>', 0 },
+    })->order_by('-confirmed')->all;
+    my $stuck_reports_count = scalar @stuck_reports;
+    my $category_count = scalar @{$params->{categories}};
+
+    my @unconfirmed_reports;
+    my $unconfirmed_reports_count;
+
+    if ($params->{unconfirmed}) {
+        @unconfirmed_reports = FixMyStreet::DB->resultset('Problem')->to_body($params->{body}->id)->search({
+            category => $params->{categories},
+            state => 'unconfirmed',
+        })->order_by('-created')->all;
```

Comment:
Repeating the stuck reports each email is fine, because we want them to have to deal with them and get them processed in some manner, just like we get them. But the unconfirmed reports will not change in most cases (they'll be people who actually didn't pay, or never got that far, etc), so I think we only want to show these once per email. So we could set a flag on each one to say it's been included in this email?

Reply:
Yeah, makes sense. Appreciated if you can do the honours if you're around already and it's quick enough.